src/builder.rs

```diff
@@ -69,6 +73,25 @@ impl SessionBuilder {
         self
     }
 
+    /// Set the timeout interval after which if no data has been received from the server, ssh
+    /// will request a response from the server (`ssh -o ServerAliveInterval`).
+    ///
+    /// This value is specified in seconds. Any sub-second duration remainder will be ignored.
+    /// Defaults to `None`.
+    pub fn server_alive_interval(&mut self, d: std::time::Duration) -> &mut Self {
+        self.server_alive_interval = Some(d.as_secs().to_string());
+        self
+    }
+
+    /// Set the number of server alive messages (see [`server_alive_interval`]) which may be sent
+    /// without ssh receiving any messages back from the server (`ssh -o ServerAliveCountMax`).
+    ///
+    /// [`server_alive_interval`]: #method.server_alive_interval
+    pub fn server_alive_count_max(&mut self, max: usize) -> &mut Self {
```

Comment:
I wonder how common this is. I'm inclined to only add `server_alive_interval`, and then only add `ServerAliveCountMax` if someone actually needs it (do you?).

Reply:
I don't think so. I've addressed your suggestions and force-pushed (since it's a small diff).